usecases/log_deployment_done_event.go

```diff
@@ -0,0 +1,92 @@
+package usecases
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/CleverseAcademy/cd-compose-deployment/constants"
+	"github.com/CleverseAcademy/cd-compose-deployment/entities"
+	"github.com/CleverseAcademy/cd-compose-deployment/utils"
+	"github.com/compose-spec/compose-go/types"
+	"github.com/docker/compose/v2/pkg/api"
+	docker "github.com/docker/docker/api/types"
+	"github.com/pkg/errors"
+)
+
+type UseCaseLogDeploymentDoneEvent struct {
+	*EventLogUseCase
+}
+
+func (u *UseCaseLogDeploymentDoneEvent) Execute(prj types.Project, deployment entities.Deployment, deployedServiceName entities.ServiceName) error {
```

Comment:
should this variable being name `serviceName`?
```suggestion
func (u *UseCaseLogDeploymentDoneEvent) Execute(prj types.Project, deployment entities.Deployment, serviceName entities.ServiceName) error {
```